manifests/controllerconfig.crd.yaml

```diff
@@ -170,6 +170,34 @@ spec:
                 description: images is map of images that are used by the controller
                   to render templates under ./templates/
                 type: object
+              imageRegistryBundleUserData:
```

Comment:
Out of curiosity, were these originally named as such in image registry operator? Or are we choosing how we want these to be named

Reply:
I chose these names, open to other suggestions... they are kind of wordy at the moment